app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def destroy
+    relationship = Relationship.find(params[:id])
+    relationship.destroy if relationship.follower == current_user
+    redirect_to people_path
+  end
+
+  def create
+    leader = User.find(params[:leader_id])
+    Relationship.create(leader: leader,
+     follower: current_user) unless current_user.not_followable?(leader)

```

Comment:
This should be styled differently, keys below each other aligned or both aligned and indented below upper line like

```ruby
Relationship.create(
  leader: leader,
  follower: current_user
) unless current_user.not_followable?(leader)
```